### ISSUE TITLE
Migrate KIE Websites to Apache Infrastructure

### ISSUE THREAD

**Issue by porcelli** [apache/incubator-kie-issues#1713]
As part of the Apache KIE graduation process, we need to migrate all project websites from Red Hat infrastructure to Apache Infrastructure. All websites should be consolidated under the kie.apache.org domain, following Apache project guidelines.

## Current Websites
- Drools: https://www.drools.org
- jBPM: https://www.jbpm.org
- OptaPlanner: https://www.optaplanner.org
- Kogito: https://kogito.kie.org
- SonataFlow: https://sonataflow.org
- KIE: https://kie.org
- Blog: http://blog.kie.org

**Comment by porcelli** [comment]
I see challenges with the current blog, as Apache has no infrastructure to host it - and it's my understanding that such a content cannot be hosted outside Apache.

**Comment by Rikkola** [comment]
@porcelli Looking at few other projects. They just have releases in the blogs. I understand that so that the project specific data and announcements need to be in the project blog. Many even lack the RSS feature.

However everyone is free to blog where and what they want. Having a page that links to bloggers should be no different than linking to commercial offerings. Even if we link to a blog of blogs that collects different blogs under it. ( Did I say blogs? )

**Comment by porcelli** [comment]
@Rikkola agreed. I think we won't be able to continue with previous consolidated blog infrastructure.

**Comment by tiagobento** [comment]
I think we still have work to do here. sonataflow.org is not migrated yet.

optaplanner.io needs to move under *.apache.org

kie.org is still up.

Including kogito.kie.org

**Comment by Rikkola** [comment]
Sonataflow has been migrated, but looks like the domain was not changed to forward like the rest. 

Same goes for kie.org, was not changed to forward. Only thing left to move is the blog, but we lack the capability for that at Apache.

https://kie.apache.org/docs/10.1.x/sonataflow/serverlessworkflow/latest/index.html

**Comment by tiagobento** [comment]
@Rikkola What's wrong with https://kie.apache.org/blog/?

**Comment by Rikkola** [comment]
@tiagobento I should rephrase. Old blog was Wordpress and current one is git based. So the blog has been moved. Moving old content is problematic.

**Comment by tiagobento** [comment]
Status update:

- Drools: https://www.drools.org/
  - Redirects to https://kie.apache.org now. I'm working on getting it to redirect to https://kie.apache.org/drools
- jBPM: https://www.jbpm.org/
  - Redirects to https://kie.apache.org now. I'm working on getting it to redirect to https://kie.apache.org/jbpm
- OptaPlanner: https://www.optaplanner.org/
  - Redirects to https://kie.apache.org now. I'm working on getting it to redirect to https://kie.apache.org/optaplanner
- Kogito: https://kogito.org/
  - Redirects to https://kie.apache.org now. I'm working on getting it to redirect to https://kie.apache.org/kogito
- Kogito: https://kogito.kie.org/
  - Still live as a separate website. Content has been migrated to https://kie.apache.org/kogito. I'm working on getting it to redirect.
- SonataFlow: https://sonataflow.org/
  - Still live as a separate website. Content has been migrated to https://kie.apache.org/docs/10.2.x/sonataflow. I'm working on getting it to redirect. SonataFlow also has a dedicated landing page on https://kie.apache.org/sonataflow
- KIE: https://kie.org/
  - Still live as a separate website. Content has been migrated to https://kie.apache.org. I'm trying to get it to redirect now.
- Blog: http://blog.kie.org/
  - Out of air as the underlying Wordpress infrastructure was not maintained. The Apache KIE blog has a new home under https://kie.apache.org/blog, but content has not been migrated yet. I'm working on it with @mdproctor.

**Comment by tiagobento** [comment]
Additionally: the DMN FEEL handbook (https://kiegroup.github.io/dmn-feel-handbook/) should have its home under https://kie.apache.org too. That's a separate effort as well.

See:

- https://github.com/apache/incubator-kie-issues/issues/702
